networking/network_observability/network-observability-overview.adoc

```diff
@@ -24,7 +24,9 @@ The Network Observability Operator requires the following Operators:
 [id="network-observability-operator"]
 == Network Observability Operator
 
-The Network Observability Operator deploys a monitoring pipeline to collect and enrich the network flows. The eBPF agent, which is deployed as a `daemonset` object, creates the network flows. The network flows are then collected and enriched with the Kubernetes metadata, before storing in Loki.
+The Network Observability Operator deploys a monitoring pipeline to collect and enrich the network flows. The eBPF agent, which is deployed as a `daemonset` object, creates the network flows. The network flows are then collected and enriched with the Kubernetes metadata, before storing in Loki. 
```

Comment:
I do not think you need a comma before "before" since that phrase should be restrictive for the sentence.